### ISSUE TITLE
Final review of faiths

### ISSUE THREAD

**Issue by jj248** [MattTheLegoman/RealmsInExile#558]
Things to do:
- [x] Move Rohirric Holy Site from Aglarond to Edoras
- [x] Extra Holy Site for Woodsmen Faith
- [x] Reduce domain taxes bonus for Gardens of Silloth from 10% to 5%
- [x] Figure out how to trigger different localization based on faith group (unify right of the hall and ritual hospitality)
- [x] Northmen god names implementation (and unique for Feastroa + Aldenan)
- [x] Add Gradens of Silloth to Feastroa
- [x] Concepts + Review of Fiskurkyn description
- [x] Rework Fiskurkyn Holy Sites
- [x] Add Sairtheod and Framsburg to Withered Court and Fey Court
- [x] Add Barad-dur to Kerkassk
- [x] Add Olbamarl to Nurniag
- [x] Swap out religious law for another tenet on Rhunnic
- [x] Broken trait in Orc faiths
- [x] Too many virtues in White Hand Cult
- [x] Add Barad-dur to Cult of the Ulukai
- [x] Comment out Dragon-cult for now (for 4.0)
- [x] Two Illuminati Faith Icons
- [x] Swap out shell for weird spider-icon for Sainbanz
- [x] Revamp Elven Faiths and reconsider Elven Tenets
- [x] Add an Elven-like tenet to Fey Court

New faiths:
- [x] Possible new faith for the Framlings
- [x] 1 extra heresy for Rhevainic
- [x] Add in Triumvirate of Araw faith for easterlings (echoes of Ainur faith, localization in files)

Potential new faith icons:
- [x] Fields of Foldewyn
- [x] Aldenan (Grape bunch)
- [x] Shakburz Icon (Chains?)
- [x] Nurniag
- [x] Cult of the Uulkai (Dark Purple Jewel)

**Comment by JukeJukestar** [comment]
Copying over from discord - my review of current faiths
### General
Where we are good:
- Ammount of new faiths
- Holy Order names (added a bunch myself lol)
- Faith placement

Where we lack:
- Proper faith descriptions
- Tenets (Especially the Leofring faiths and Northmen faith)
- God names? (Currently most use the regular names for the valar instead of the northmen ones) 
-  Holy Site placement and effects
### Holy Sites as a focus
I think a big thing that is often overlooked with holy site placement is its importance for unreformed faiths
with organized faiths its not that important as the small buffs you get for controlling them arent too important
so for certain faiths you can spread them around the map to important locations, but for unreformed faiths, 2-3 should be in their homelands with the other two a bit further away so its possible to reform them without conquering across the map.

Negative prime example are the Daen Coentis, who have their holy sites spread out in Enedwaith and have to conquer their gwaithuirim cousins to reform (Which sucks if you want to ally with them, as we tried in the Middle Men MP)
at least 2 should be in pinnath gelin, with another in their lands east of there (controlled by Gondor and maybe two more in Andrast and Greater Gondor.

TL;DR: Add 5 holy sites for unreformed, place 3 in their sphere of influence, dont force them to conquer areas they shouldnt have to.

**Comment by VectorMaximus** [comment]
Things to do:
- [ ] Move Rohirric Holy Site from Aglarond to Edoras
- [ ] Extra Holy Site for Woodsmen Faith
- [ ] Reduce domain taxes bonus for Gardens of Silloth from 10% to 5%
- [ ] Figure out how to trigger different localization based on faith group (unify right of the hall and ritual hospitality)
- [ ] Northmen god names implementation (and unique for Feastroa + Aldenan)
- [ ] Add Gradens of Silloth to Feastroa
- [ ] Concepts + Review of Fiskurkyn description
- [ ] Rework Fiskurkyn Holy Sites
- [ ] Add Sairtheod and Framsburg to Withered Court and Fey Court
- [ ] Reconsider Elven Tenets, consider adding an Elven-like tenet to Fey Court
- [ ] 1 extra heresy for Rhevainic
- [ ] Add Barad-dur to Kerkassk
- [ ] Add Olbamarl to Nurniag
- [ ] Swap out religious law for another tenet on Rhunnic
- [ ] Broken trait in Orc faiths
- [ ] Too many virtues in White Hand Cult
- [ ] Add Barad-dur to Cult of the Ulukai
- [ ] Comment out Dragon-cult for now (for 4.0)
- [ ] Rethink Elven Faiths

Considerations:
- [ ] Possible new faith for the Framlings
- [ ] Add in Triumvirate of Araw faith for easterlings (echoes of Ainur faith, localization in files)

Potential new faith icons:
- [ ] Fields of Foldewyn
- [ ] Aldenan (Grape bunch)
- [ ] Shakburz Icon (Chains?)
- [ ] Nurniag
- [ ] Cult of the Uulkai (Dark Purple Jewel)

**Comment by RedArkady** [comment]
Re the third holy site for the woodmen, I'd probably put that in the East Bight for historic reasons, and to buff the eastern woods a bit (and make them a target)

**Comment by RedArkady** [comment]
Vector, I think you also mentioned making Amarthrynd a silvan holy site, as per the special building description.

**Comment by RedArkady** [comment]
And (sorry for the message spam) I do think it makes sense to give the Famlings their own faith, make the Leofring faith heresies of it, and the Rohirric faith a reformed version?

**Comment by VectorMaximus** [comment]
> Vector, I think you also mentioned making Amarthrynd a silvan holy site, as per the special building description.

Already done for the Daetha Tauron (Nandor/Silvan) faith.

**Comment by jj248** [comment]
All done.  Closing.
